client/me/help/help-contact-closed/index.jsx

```diff
@@ -7,31 +7,30 @@
 import React from 'react';
 import i18n, { localize } from 'i18n-calypso';
 
+const christmas2017ClosureStartsAt = i18n.moment( 'Sun, 24 Dec 2017 00:00:00 +0000' );
```

Comment:
Would it be easier to use generic variable names and just change their values?

Reply:
I could go either way on this. I like explicitly stating the reason for the closure because it's clearer later on whether you should expect the form to be affected (e.g. in April you see `christmas2017ClosureStartsAt` and can quickly tell "no this shouldn't affect anything right now"). Sometimes the logic gets more complicated as well where we need more than just two variables.